sparse/client.go

```diff
@@ -127,7 +130,7 @@ func (client *syncClient) syncFileContent(file FileIoProcessor, fileSize int64)
 	return nil
 }
 
-func (client *syncClient) sendHTTPRequest(method string, action string, interval Interval, data []byte) (*http.Response, error) {
+func (client *syncClient) sendHTTPRequest(method string, action string, interval Interval, queries map[string]string, data []byte) (*http.Response, error) {
```

Comment:
I mean `func (client *syncClient) sendHTTPRequest(method string, action string, queries map[string]string, data []byte) (*http.Response, error)`

All callers need to put `interval.Begin` and `interval.End` in `queries` before calling this function. Then This function can blindly put all entries of `queries` in `req.URL.Query()` without `directIO, err := queries["directIO"]`

Reply:
updated